.github/workflows/release-build.yml

```diff
@@ -291,3 +291,21 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         UPLOAD_URL: ${{ github.event.release.upload_url }}
       run: ./.github/workflows/upload-release-asset.sh "artifacts/Helics-$(git rev-parse --abbrev-ref "${GITHUB_REF}")-SHA-256.txt"
+
```

Comment:
Is there a corresponding receiver in the other repos?

Reply:
The git diff link in the PR description goes to one of them.